selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/ImportProjectFromZipTest.java

```diff
@@ -98,11 +96,24 @@ public void importProjectFromZipTest() {
     // possible to read logs in case of test failure
     testWorkspace = testWorkspaceProvider.getWorkspace(WORKSPACE, defaultTestUser);
 
-    seleniumWebDriverHelper.switchToIdeFrameAndWaitAvailability();
-    toastLoader.waitToastLoaderAndClickStartButton();
-    ide.waitOpenedWorkspaceIsReadyToUse();
-    explorer.waitItem(PROJECT_NAME);
-    explorer.waitAndSelectItem(PROJECT_NAME);
-    explorer.openContextMenuByPathSelectedItem(PROJECT_NAME);
+    theiaIde.switchToIdeFrame();
+    theiaIde.waitTheiaIde();
+    theiaIde.waitLoaderInvisibility();
+    theiaIde.waitTheiaIdeTopPanel();
+    theiaProjectTree.waitFilesTab();
+
+    // wait the project in the tree
+    theiaProjectTree.clickOnFilesTab();
+    theiaProjectTree.waitProjectsRootItem();
+    theiaProjectTree.waitProjectAreaOpened();
+    theiaIde.waitTheiaIdeTopPanel();
+    theiaProjectTree.waitFilesTab();
+
+    try {
+      theiaProjectTree.waitItem(PROJECT_NAME);
+    } catch (TimeoutException ex) {
+      // remove try-catch block after issue has been resolved
+      fail("Known permanent failure https://github.com/eclipse/che/issues/13579");
```

Comment:
If there is permanent failure, let's set `@Test(groups = UNDER_REPAIR)` for the test to exclude it from regular test execution.

Reply:
OK.